app/templates/ui/styles/theme.less

```diff
@@ -208,3 +208,7 @@ input.vertical {
   height: 100px;
   writing-mode: bt-lr;
 }
+
+.padtop {
+  padding-top: 30px;
+}
```

Comment:
In relation to my earlier padtop comment: I think Jen also applied a padding fix for dashboard theme. Sure this is necessary?